lib/gc/Target/LLVM/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+gc_add_mlir_dialect_library(MLIRGENTarget
+  GEN/Target.cpp
+
+  OBJECT
+
+  ADDITIONAL_HEADER_DIRS
+  ${MLIR_MAIN_INCLUDE_DIR}/mlir/Dialect/LLVMIR
+  ${PROJECT_SOURCE_DIR}/include/gc/Dialect/LLVMIR
+
+  LINK_LIBS PUBLIC
+  MLIRIR
+  MLIRExecutionEngineUtils
+  MLIRSupport
+  MLIRGPUDialect
+  MLIRTargetLLVM
+  LLVMSPIRVCodeGen
```

Comment:
This is weird, I couldn't reproduce the linking problem locally and I get a runtime problem with static llvm options reinitialization when I include `LLVMSPIRVCodeGen` as a dependency.